src/singer_clojure/parse.clj

```diff
@@ -23,7 +23,7 @@
 (defn state
   "This function exists as a test seam and for the post condition"
   [state-file]
-  {:post [(is (map? %) (format "map %s is not valid" %))]}
+  {:post [(is (map? %) (format "state-file %s is not a map" %))]}
```

Comment:
Might be good to use the name here. I didn't even realize the `%` version of the parameter would work here.

Reply:
Sure

Reply:
So % is the return value so we're trying to figure out how to refer to that in a named way, because the message is wrong, it's not that state-file is a map, but slurp-json of state-file is

Reply:
Ahhh, okay, that makes sense. :+1: